test/features/proof_of_address.feature

```diff
@@ -0,0 +1,151 @@
+@browser
+Feature: Proof of address
+
+  Scenario: Test file upload for bank statement
+    Given I navigate to the SDK with PoA feature enabled
+    When I click on primary_button (SDK)
+    When I click on start_verification_button (POA)
+    When I click on benefits_letter_cell (POA)
+    When I click on continue_button (POA)
+    When I upload national_identity_card
```

Comment:
Can you add a comment explaining why you are uploading a national_identity_card when the step is called `Test file upload for bank statement`

Reply:
Or you could wrap it into a step called `When I upload any document` and you can use it for all the other scenarios